src/cpp/shared/adventure/hero.cpp

```diff
@@ -49,7 +49,7 @@ signed __int8 captainStats[MAX_FACTIONS][4] = {
   {0, 0, 0, 0},
   {0, 0, 0, 0},
   {0, 0, 0, 0},
-  {1, 1, 1, 1}
+  {1, 1, 1, 1}  // TODO: is Cyborg a might or magic hero?
```

Comment:
That's a good question.

They should probably be considered a magic hero, since we're developing a lot of custom spells for them.